numpy/__init__.py

```diff
@@ -155,6 +155,15 @@
     from . import matrixlib as _mat
     from .matrixlib import *
 
+    # Deprecations introduced in 1.25.0, 2023-05-xx TODO update day
```

Comment:
This should now be 1.26.0, since the `maintenance/1.25.x` branch just got created. The plan is to do a 1.26.0 release in September or so, with a number of deprecations and with the build system changes needed for Python 3.12 support.

Reply:
Sure, changed!